config/initializers/money.rb

```diff
@@ -1 +1,2 @@
 Money.rounding_mode = BigDecimal::ROUND_HALF_EVEN
+Money.default_currency = Money::Currency.new('USD')
```

Comment:
Shouldn't this be default currency of the instance?

Reply:
Oops, yes. Good catch 😅